database/queries/addToCart.js

```diff
@@ -0,0 +1,16 @@
+const { pool } = require('../index');
+
+const addToCart = (params, callback) => {
+  const { userToken, sku_id, active } = params;
+  const queryString = `INSERT INTO cart (user_session, product_id, active) VALUES (${userToken}, ${sku_id}, ${active})`;
```

Comment:
To prevent SQL injection, you should use ($1, $2, $3) instead of template literals, then pass params in as the second argument of your query